### ISSUE TITLE
Add SASS/Compass support

### ISSUE THREAD

**Issue by emcpadden** [ngbp/ngbp#31]
First off, let me say that I really like this kickstarter.  It is a great stating point.  I think it's great that it has the LESS support. 

It would be great if it also supported SASS/Compass.

**Comment by joshdmiller** [comment]
@emcpadden That's not a bad idea. I'm no fanboy, I'm just used to LESS - honestly, mostly because it's Twitter's default. :-)

Do you have any ideas on how to accomplish this while avoiding having to maintain two branches or two repos?

**Comment by MikeFielden** [comment]
Hey Josh,
I didnt see this issue before right now, but I forked this repo and made it use Compass. I used your same structure for most things and kept your awesome comments and most everything else. 

That repo can be found [here](https://github.com/MikeFielden/ng-compass-boilerplate)
I didn't mean to step on your toes at all. 

-Mike

**Comment by nullbio** [comment]
I agree, native SASS/Compass support would be wonderful! Not sure how you'd go about adding this without a separate branch/repo though.

I'll give your version a try Mike, thanks for the link.

**Comment by joshdmiller** [comment]
**Update:** I have an idea how we can do this without maintaining a separate branch or repo. Basically, we'd use a custom task to call either the LESS or SASS compilation task based on a flag specified in the Gruntfile. I'm busy pushing out v0.3 right now, but I should be able to take a whack at this in the next few weeks. Otherwise, PRs are always welcome. :-)

**Comment by astronaute77** [comment]
I would like to see SASS & Compass support too :8ball:

**Comment by HughxDev** [comment]
Any update on this? And/or does anybody know a method for switching from Less to Sass?

**Comment by julkue** [comment]
Possibly duplicate of https://github.com/ngbp/ngbp/issues/191
